app/src/main/java/co/develhope/chooseyourownbeer/network/PunkService.kt

```diff
@@ -0,0 +1,9 @@
+package co.develhope.chooseyourownbeer.network
+
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface PunkService {
+    @GET("{beers}")
+    suspend fun listRepos(@Path("beers") user: String?): BeersResult
```

Comment:
user? Se ci aspettiamo una lista di birre non credo c'entri nulla lo user. Non credo che sia necessario questo parametro, mi sembra che non esista il caso in cui questo valore possa cambiare, se è così non lo metterei nel costruttore ma lo userei direttamente tramite una costante

Reply:
Questo non so come cambiarlo, non ho capito come usarlo tramite costante